src/auro-popover.js

```diff
@@ -102,8 +102,14 @@ class AuroPopover extends LitElement {
     if (this.sticky) {
       this.trigger.addEventListener('click', handleShow);
     } else {
-      this.trigger.addEventListener('mouseenter', handleShow);
-      this.trigger.addEventListener('mouseleave', handleHide);
+
+      if(this.trigger.parentElement.nodeName === 'AURO-POPOVER') {
```

Comment:
This check seems weird, but I'm not sure if there's a better way.

Reply:
Agree, I couldn't think of another way to check that the trigger was not detached.